packages/jupyter-ai/jupyter_ai/chat_handlers/export.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, *args, **kwargs):
 
     def chat_message_to_markdown(self, message):
         if isinstance(message, AgentChatMessage):
-            return f"**Agent**: {message.body}"
+            Agent = self.config_manager.persona.name
+            return f"**{Agent}**: {message.body}"
```

Comment:
```suggestion
            agent = self.config_manager.persona.name
            return f"**{agent}**: {message.body}"
```

Python conventions typically favor lowercase letters to start a variable like this one.